apps/dav/lib/Controller/ExampleContentController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\DAV\Controller;
+
+use OCA\DAV\AppInfo\Application;
+use OCP\App\IAppManager;
+use OCP\AppFramework\ApiController;
+use OCP\AppFramework\Http;
+use OCP\AppFramework\Http\JSONResponse;
+use OCP\Files\AppData\IAppDataFactory;
+use OCP\Files\IAppData;
+use OCP\Files\NotFoundException;
+use OCP\IConfig;
+use OCP\IRequest;
+
+class ExampleContentController extends ApiController {
+
+	/** @var IAppData */
+	private $appData;
+	public function __construct(
+		IRequest $request,
+		private IConfig $config,
+		private IAppDataFactory $appDataFactory,
+		private IAppManager $appManager,
+	) {
+		parent::__construct(Application::APP_ID, $request);
+		$this->appData = $this->appDataFactory->get('dav');
+	}
+
+
+	/**
+	 * update appconfig (admin setting)
+	 *
+	 * @param string allow the value to set
+	 *
+	 * @return JSONResponse an empty JSONResponse with respective http status code
+	 */
+	public function setAppConfig($key, $allow) {
```

Comment:
```suggestion
	public function setEnableDefaultContact($allow) {
```

Nit: I don't like having a general app config route here. Perhaps it could be specialized to only set the default contact setting for now.

Reply:
I did it in case you want to rebase the calendar content on top, I'm okay with changing it